projects/FHC-AtmelStudio/Fussl-01/TestOnTargetDevice/f_static_task_runner.h

```diff
@@ -0,0 +1,96 @@
+/**
+@file f_static_task_runner.h
+@author Maximilian Starke
+@date 2019
+@copyright All rights reserved. For any license please contact the author.
+*/
+
+#ifndef F_STATIC_TASK_RUNNER_H_
```

Comment:
Add an action item to maybe put the static_task_runner into fsl standard library, not only in test framework